src/App.jsx

```diff
@@ -1,7 +1,27 @@
 import { useState } from 'react'
+import { Route, Routes } from 'react-router-dom';
+
+import Navbar from './components/Navbar';
+import Articles from './components/Articles';
+import Topics from './components/Topics';
+import Profile from './components/Profile';
+import Users from './components/Users';
 
 function App() {
-  
+	const [currentUser, setCurrentUser] = useState({});
+
+	return (
+		<div>
+			<h1>NC-News</h1>
+			<Navbar />
+			<Routes>
+				<Route path="/" element={<Articles />} />
+				<Route path="/topics" element={<Topics />} />
+				<Route path="/profile" element={<Profile />} />
+				<Route path="/users" element={<Users />} />
```

Comment:
There's no need for these routes yet - please make sure you're only working on one ticket at a time!!